### DISCUSSION TITLE
Possibility to "manage team admins" when GitHub team is connected to an IdP

### DISCUSSION THREAD

**Discussion by TiagoJVO** [community/community#39072] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi everyone!

Setting up my Company's GitHub Teams to be linked to our internal IdP to facilitate team changes, I faced a problem that I cannot find the answer to in any public documentation or discussion.

So, I went to the settings of the team, and under "Identity Provider Groups" I selected the appropriate group. This syncs the team memberships from the external group, perfect!
The problem is that by doing this, I completely lose the capability of managing "permissions" for this team, for example, I want the Team Leader to be an "administrator" of this particular team. 
The perfect way to do this, would be to import that definition from the external group provider, and admins of that group are also set as "team admins", but I don't think that is possible... But at least there should be a way to change the permission of the team members on GitHub, but the dropdown to do that under "members" completely disappeared as soon as the team got synced from the IdP.

Plus, it seems that it kept the "administrators" that were set prior to IdP Setup, and this is concerning.

Does anyone have context on this use case?

Cheers!

**Comment by dharanishmallelaKGS** [comment]
Still no Answer
